pkgs/development/python-modules/radian/default.nix

```diff
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "radian";
-  version = "0.6.12";
+  version = "0.6.13";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

Comment:
This needs to be bumped up to `pythonOlder "3.9"`.

Reply:
Why is that? in radian's requirements I see that Python 3.7 or above is needed

Reply:
Ah, I guess this is why?

https://github.com/NixOS/nixpkgs/issues/312288

Reply:
Pretty much. The author also tests it on 3.9.